modules/account-lib/src/coin/eth/iface.ts

```diff
@@ -1,17 +1,31 @@
 import { BaseFee } from '../baseCoin/iface';
 import { KeyPair } from './keyPair';
 
-export interface Fee extends BaseFee {
+export interface EthFee extends BaseFee {
```

Comment:
nit: Why is this called EthFee now? nothing has changed, it still lives in the eth controller